core/dispatcher/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -483,19 +484,18 @@ class ContainerPool(
         val key = makeKey(action, auth)
         val warmedContainer = if (limits.memory == defaultMemoryLimit && imageName == nodeImageName) getWarmNodejsContainer(key) else None
         val containerName = makeContainerName(action)
-        val con = warmedContainer getOrElse makeGeneralContainer(key, containerName, imageName, limits)
+        val con = warmedContainer getOrElse makeGeneralContainer(key, containerName, imageName, limits, action.exec.isInstanceOf[BlackBoxExec])
```

Comment:
Not sure what's cheaper, but `action.exec.kind == Exec.blackbox` would be another solution.